src/imitation/algorithms/adversarial/common.py

```diff
@@ -429,9 +447,18 @@ def train(
                 callback(r)
             self.logger.dump(self._global_step)
 
+    @overload
+    def _torchify_array(self, ndarray: np.ndarray) -> th.Tensor:
+        ...
+
+    @overload
+    def _torchify_array(self, ndarray: None) -> None:
+        ...
+
     def _torchify_array(self, ndarray: Optional[np.ndarray]) -> Optional[th.Tensor]:
         if ndarray is not None:
             return th.as_tensor(ndarray, device=self.reward_train.device)
+        return None
```

Comment:
Does `mypy` insist on this? Kind of sad (no return should be semantically equivalent to returning None and I think is more Pythonic)

Reply:
I can add a type ignore or customize the errors in the config file, but yes, by default it wants you to have explicit returns for legibility. I tend to prefer the more pythonic way, but I don't disagree it's more readable.

Reply:
OK, fine to leave as-is, better than a type ignore.